vhdbuilder/packer/post-install-dependencies.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+OS=$(sort -r /etc/*-release | gawk 'match($0, /^(ID_LIKE=(coreos)|ID=(.*))$/, a) { print toupper(a[2] a[3]); exit }')
+UBUNTU_OS_NAME="UBUNTU"
+
+source /home/packer/provision_installs.sh
+source /home/packer/provision_installs_distro.sh
+source /home/packer/provision_source.sh
+source /home/packer/provision_source_distro.sh
+source /home/packer/tool_installs.sh
+source /home/packer/tool_installs_distro.sh
+
+CPU_ARCH=$(getCPUArch)  #amd64 or arm64
+VHD_LOGS_FILEPATH=/opt/azure/vhd-install.complete
+
+if [[ $OS == $UBUNTU_OS_NAME ]]; then
+  # remove apport
+  apt-get purge --auto-remove apport open-vm-tools -y
+fi
+
+# shellcheck disable=SC2129
+echo "kubelet/kubectl downloaded:" >> ${VHD_LOGS_FILEPATH}
+ls -ltr /usr/local/bin/* >> ${VHD_LOGS_FILEPATH}
+
+# shellcheck disable=SC2010
+ls -ltr /dev/* | grep sgx >>  ${VHD_LOGS_FILEPATH} 
+
+echo -e "=== Installed Packages Begin\n$(listInstalledPackages)\n=== Installed Packages End" >> ${VHD_LOGS_FILEPATH}
+
+echo "Disk usage:" >> ${VHD_LOGS_FILEPATH}
+df -h >> ${VHD_LOGS_FILEPATH}
+
+
+# warn at 75% space taken
+[ -s $(df -P | grep '/dev/sda1' | awk '0+$5 >= 75 {print}') ] || echo "WARNING: 75% of /dev/sda1 is used" >> ${VHD_LOGS_FILEPATH}
+# error at 99% space taken
+[ -s $(df -P | grep '/dev/sda1' | awk '0+$5 >= 99 {print}') ] || exit 1
+
+echo "Using kernel:" >> ${VHD_LOGS_FILEPATH}
+tee -a ${VHD_LOGS_FILEPATH} < /proc/version
+{
+  echo "Install completed successfully on " $(date)
+  echo "VSTS Build NUMBER: ${BUILD_NUMBER}"
+  echo "VSTS Build ID: ${BUILD_ID}"
+  echo "Commit: ${COMMIT}"
+  echo "Ubuntu version: ${UBUNTU_RELEASE}"
+  echo "Hyperv generation: ${HYPERV_GENERATION}"
+  echo "Feature flags: ${FEATURE_FLAGS}"
+  echo "Container runtime: ${CONTAINER_RUNTIME}"
+  echo "FIPS enabled: ${ENABLE_FIPS}"
+} >> ${VHD_LOGS_FILEPATH}
+
+if [[ $(isARM64) != 1 ]]; then
+  # no asc-baseline-1.1.0-268.arm64.deb
+  installAscBaseline
+fi
+
+if [[ ${UBUNTU_RELEASE} == "18.04" || ${UBUNTU_RELEASE} == "22.04" ]]; then
+  if [[ ${ENABLE_FIPS,,} == "true" || ${CPU_ARCH} == "arm64" ]]; then
+    relinkResolvConf
```

Comment:
this...is odd. not related to you. but why is this conditioned on arm or fips?

should be done for all of 18, 20, and 22.04. probably already done elsewhere...